### PR TITLE
Week 5 assignment

### PR DESCRIPTION
**Pull request by genjuro86**
It was my first trial of week 5 assignment.

Finished week 4 assignment, and Q1 of week 5 assignment.
I was stuck at Q2 of week 5 assignment, because there are unmatched values...
I will figure it out later.

### CODE REVIEW

File: Individual Assignments/Nam Young-Ju/Week 5/W05-assignment(NYJ).do [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+replace department = substr(department, 15, .)
+replace department = lower(department)
+
+tempfile CIV_density
+save `CIV_density'
+
+merge 1:m department using `CIV_data'
+
+rename DENSITEAUKM density
+ * there are still 12,434 not matched results. I'll figure out.
```
Comment (XZXinyuZheng) [inline Individual Assignments/Nam Young-Ju/Week 5/W05-assignment(NYJ).do:81]: There are three reasons for the unmatching: 
(1) The column of the department extracted from excel contains leading or ending spaces. If we cut those spaces out by strtrim(), we'll be able to match 12827 of them.
(2) One department name called "arrah" in excel but "arrha" in dta
(3) No department "gbeleban" in excel

File: Individual Assignments/Nam Young-Ju/Week 4/W04-assignment(NYJ).do [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+ 
+label values Table_Number Distribution 
+rename Table_Number Distribution 
+replace Age_Group = subinstr("OVERALL18 AND ABOVE", "OVERALL18", "18", .)
+* above commands are to cleaning dataset to easily figure out the table.
+
```
Comment (XZXinyuZheng) [inline Individual Assignments/Nam Young-Ju/Week 4/W04-assignment(NYJ).do:98]: It seems that there are some problems with the last three columns. Specifically, they contain special symbols such as "-" and "...".

**Review by XZXinyuZheng — COMMENTED** [review]
Hi Yong-Ju, nice work on both updated assignment04 and assignment05! Please see specific comments below the codes. Hope those comments help a little bit and hope you go well with the rest of the questions.

File: Individual Assignments/Nam Young-Ju/Week 5/W05-assignment(NYJ).do (hunk 1 above)
Comment (genjuro86) [reply Individual Assignments/Nam Young-Ju/Week 5/W05-assignment(NYJ).do:81]: Thank you Xinyu Zheng!!!

**Pull request closed without merging**